test/parallel/test-http-outgoing-proto.js

```diff
@@ -12,3 +12,63 @@ assert.strictEqual(
   typeof ClientRequest.prototype._implicitHeader, 'function');
 assert.strictEqual(
   typeof ServerResponse.prototype._implicitHeader, 'function');
+
+// validateHeader
+assert.throws(common.mustCall(() => {
```

Comment:
There's no need to use `common.mustCall()` on functions passed to `assert.throws()`